### PR TITLE
Update pushpull.yaml

### PR DESCRIPTION
**Pull request by gerard33**
Use hacs/integration/action@main instead of hacs/integration/action@master

### CODE REVIEW

**Comment by ludeeus** [comment]
Yo!
Use hacs/action instead

**Comment by gerard33** [comment]
Thx, probably an old error message shown then. I already found it strange that I got other errors now 😉

**Comment by ludeeus** [comment]
"hacs/integration/action" has not been updated in a loong time (since I makde hacs/action), looking at the error, it looks like I can just remove it, it does not work for anyone anymore, since things has changed.

File: .github/workflows/pushpull.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: "actions/checkout@v2"
 
       - name: HACS validation
-        uses: "hacs/integration/action@master"
+        uses: "hacs/action"
```
Comment (ludeeus) [inline .github/workflows/pushpull.yaml:19]:
```suggestion
        uses: "hacs/action@main"
```
Branch/tag is required

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: .github/workflows/pushpull.yaml (hunk 1 above)
Comment (gerard33) [reply .github/workflows/pushpull.yaml:19]: I was just comparing the content with my other custom integration (https://github.com/custom-components/media_player.braviatv_psk/blob/master/.github/workflows/pushpull.yaml) so will make some more updates to this file, as I have updated the bravia integration more recently.

**Comment by gerard33** [comment]
Thx @ludeeus, added the issue tracker as well and all looking good now again 😄

**Comment by ludeeus** [comment]
uuups :smile:  too early for mee, guess I need more coffee :coffee: 
Looking good now! :tada:

**Pull request merged**